my-app/src/actions.js

```diff
@@ -0,0 +1,6 @@
+const CLICK_ON_USE_CASE = 'CLICK_ON_USE_CASE';
+const CLICK_ON_IMAGE = 'CLICK_ON_IMAGE';
+const CHANGE_USE_CASE_BER_COLLOR = 'CHANGE_USE_CASE_BER_COLLOR';
+const FETCH_PHOTOS_FROM_CMPAIN_ID = 'FATCH_PHOTOS_FROM_CMPAIN_ID';
+const FETCH_USE_CASES = 'FATCH_USE_CASES'
+const CHANGE_URL_FOR_USE_CASES = 'CHANGE_URL_FOR_USE_CASES'
```

Comment:
Those are not actions, those are logic parts that should be applied in your reducers.